include/ultra64/abi.h

```diff
@@ -294,13 +294,20 @@ typedef short ENVMIX_STATE[40];
         _a->words.w1 = (u32)(a2);                                    \
 }
 
-#define aClearBuffer(pkt, d, c)                                         \
-{                                                                       \
-        Acmd *_a = (Acmd *)pkt;                                         \
-                                                                        \
-        _a->words.w0 = _SHIFTL(A_CLEARBUFF, 24, 8) | _SHIFTL(d, 0, 24); \
-        _a->words.w1 = (u32)(c);                                        \
-}
+/*
+ * Clears DMEM data by writing zeros.
```

Comment:
would it make more sense to just say "clears DMEM by writing zeros"
dmem is the physical place where the data goes, so it sounds better to me to say that the thing itself is getting cleared, instead of the data that is on it. but could be wrong

Reply:
I just copied this line from sm64 but yeah don't need the data